### PR TITLE
LVNETZZZ-17. Created settings screen

### PR DESCRIPTION
**Pull request by YarishPetro**
## Description (Include context, screenshots, ticket links etc.)
51574557_375019533319365_1206439468584665088_n
51597920_598610847268151_5765752312207245312_n
51385340_2420878288182522_1399170452340867072_n

## PR Submission Reminders:
- Include a jira ticket with number and description*
- Please keep the PR as small as possible, reviewer should be able to review in less than 15 minutes
- Check if your PR havs no breaking changes

### CODE REVIEW

File: Actions/settingsActions.js [hunk 1]
```diff
@@ -44,14 +49,21 @@ export const loadSettings = deviceId => {
...
       headers: {
         "Content-Type": "application/json"
       },
       body: JSON.stringify(newSettings)
     })
-      .then(res => res.json())
+      .then(res => {
+        if(!res.ok)
+        {
+          throw Error(res.statusText);
```
Comment (vtorkalo) [inline Actions/settingsActions.js]: don't throw error
create action storeSettingsFail and set information about error in state

File: Actions/settingsActions.js [hunk 2]
```diff
@@ -44,14 +49,21 @@ export const loadSettings = deviceId => {
...
+        }
+        return res.json();
```
Comment (vtorkalo) [inline Actions/settingsActions.js]: return of res not needed

File: Screens/SettingsScreenComponents/ChooseLanguage.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import LocalizeComponent from '../../Localization/LocalizedComponent';
+
+class ChooseLanguage extends LocalizeComponent {
+    state = {
+        settings: { language: "" }
+    }
+
+    componentDidUpdate(prevState) {
+        if (!prevState.settings.error && this.props.settings.error) {
+            Alert.alert(this.t('Error!'), this.t('Please try again'));
```
Comment (vtorkalo) [inline Screens/SettingsScreenComponents/ChooseLanguage.js]: i think that displaying of error in some red validation box will be better

Comment (YarishPetro) [reply Screens/SettingsScreenComponents/ChooseLanguage.js]: Good idea

**Review by YarishPetro — COMMENTED** [review]
51754631_544106942751387_1821190039015522304_n
51511139_334523570514095_3997879511872962560_n

**Review by vtorkalo — APPROVED** [review]

**Pull request merged**